requirements.txt

```diff
@@ -10,7 +10,6 @@ importlib_metadata>=3.6  # The "selectable" entry points were introduced in `imp
 jmespath>=0.9.5, <1.0
 pip-tools~=6.5
 pluggy~=1.0.0
-python-json-logger~=2.0
```

Comment:
What was this requirement used for?

Reply:
It was for an old style of logging format that existed before we switched to rich logging. It should have been removed before, just we missed it.